tests/core/test_event_form.py

```diff
@@ -83,3 +84,13 @@ def test_participating_users_can_see_otherwise_invisible_event(
 
     # check that we can get (200 OK) the event details as a participant
     assert django_app.get(event.get_absolute_url(), user=volunteer)
+
+
+def test_management_group_can_make_event_visible_for_all_groups(
+    django_app, service_event_type, manager, groups
+):
+    event_form = django_app.get(
+        reverse("core:event_create", kwargs=dict(type=service_event_type.pk)),
+        user=manager,
+    ).form
+    assert len(Group.objects.all()) == len(event_form["visible_for"].options)
```

Comment:
```suggestion
    assert set(Group.objects.all()) == set(event_form["visible_for"].options)
```

Reply:
E       AssertionError: assert {<Group: Managers>, <Group: Planners>, <Group: Volunteers>} == {('2', True, 'Planners'), ('3', True, 'Volunteers'), ('1', True, 'Managers')}

Reply:
ah whoopsie of course. I'd still like to check that semantically, the groups are the same